guide.md

```diff
@@ -0,0 +1,18 @@
+[Back](./README.md)
+
+# Guide
+
+1. The list is sorted with alphabetical sections. With exceptions for "Numbers", "Symbols" and "Others" which are special.
+2. In section, abbrs are sorted in alphabetical order.
+3. Abbrs are in word-abbr format.
+4. Colored circles before the abbr mean the degree of recommendation:
+
+-  🟢 Recommended
+-  🟡 Context-sensitive
+-  🔴 Not recommended
+
+Unrecommended abbrs can still be considered.
+The most important thing is to remain consistent in your code.
+
+4. Context abbrs are more suitable in certain cases. The context is specified between { } after the abbr.
```

Comment:
We can add this next to :yellow_circle:

Reply:
In the contributing?

Reply:
I've still followed my proposal: we need to add only :green_circle:  :yellow_circle:  and :red_circle:  in the README, with one sentence explanation for each. And we can explain there what `{ }` means in case of :yellow_circle: